source/js/storage.js

```diff
@@ -0,0 +1,189 @@
+const userInfo = {};
+const renderedText = {};
+
+window.addEventListener("DOMContentLoaded", init);
+
+// temporary init for demo
+async function init() {
+  getUserInfo().then(renderText);
+  bindButton();
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 4 spaces but found 2.](https://app.codacy.com/gh/cse110-fa21-group27/cse110-fa21-group27/pullRequest?prid=8429666)
```suggestion
    bindButton();
```